samples/outlook/preview-apis-signature/work-with-client-signatures.yaml

```diff
@@ -0,0 +1,115 @@
+            } else {
+              console.error(asyncResult.error);
+            }
+          });
+        }
+
+        function getComposeType() {
+          // Get the compose type of the current item (compose only)
+          Office.context.mailbox.item.getComposeTypeAsync(function(asyncResult) {
+            if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+              console.log(
+                "getComposeTypeAsync succeeded with composeType: " +
+                  asyncResult.value.composeType +
+                  " and coercionType: " +
+                  asyncResult.value.coercionType
+              );
+            } else {
+              console.error(asyncResult.error);
+            }
+          });
+        }
+
+        function setSignature() {
+          // Set the signature for the current item
+          var signature = $("#signature").val();
+          console.log(`Setting signature to "${signature}".`);
+          Office.context.mailbox.item.body.setSignatureAsync(signature, function(asyncResult) {
+            if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+              console.log("setSignatureAsync succeeded");
+            } else {
+              console.error(asyncResult.error);
+            }
+          });
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to check if the client signature is enabled, disable the client signature, get the compose type, and set the signature.</p>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <button id="isClientSignatureEnabled" class="ms-Button">
+            <div class="ms-Button-label">isClientSignatureEnabled</div>
+          </button>
+            <button id="disableClientSignature" class="ms-Button">
+            <div class="ms-Button-label">disableClientSignature</div>
+          </button>
+            <button id="getComposeType" class="ms-Button">
+            <div class="ms-Button-label">getComposeType</div>
+          </button>
+            <div class="ms-TextField">
+                <label class="ms-Label">Signature:</label>
+                <input id="signature" class="ms-TextField-field" type="text" value="This is my signature" placeholder="">
+                  </div>
+                <button id="setSignature" class="ms-Button">
+                    <div class="ms-Button-label">setSignature</div>
+            </button>
+        </section>
+    language: html
+style:
+    content: |-
+        section.samples {
+            margin-top: 20px;
+        }
+
+        section.samples .ms-Button, section.setup .ms-Button {
+            display: block;
+            margin-bottom: 5px;
+            margin-left: 20px;
+            min-width: 80px;
+        }
+    language: css
+libraries: |
+    https://appsforoffice.microsoft.com/lib/beta/hosted/office.js
```

Comment:
When the APIs are available in beta, let me know so I can add details to the office-js-preview d.ts in DefinitelyTyped

Reply:
It is not on the beta CDN yet. I will let you know when it is deployed.

Reply:
These APIs are now available in beta.

Reply:
These APIs are now available in beta